modules/web-console/web-console-server/src/test/java/org/apache/ignite/console/AbstractSelfTest.java

```diff
@@ -34,11 +38,15 @@
 @SpringBootTest(webEnvironment = RANDOM_PORT, classes = {TestGridConfiguration.class})
 @DirtiesContext
 public class AbstractSelfTest {
+    /** Ignite. */
+    @Autowired
+    private Ignite ignite;
+
     /**
      * @throws Exception If failed.
      */
     @BeforeClass
-    public static void setup() throws Exception {
+    public static void tearUp() throws Exception {
```

Comment:
The setUp is more preferred than tearUp (I did't see that anyone use such wording)

Reply:
Rename to `beforeTests`